requirements/dev_requirements.txt

```diff
@@ -2,6 +2,7 @@ accessible-pygments==0.0.4
     # via pydata-sphinx-theme
 aiohappyeyeballs==2.4.0
     # via aiohttp
+aiohttp==3.10.11
```

Comment:
This merge commit messed this up a bit. Same for the vulnerabilities notebook, but that is now fixed.

Reply:
Yep, sorry for that. I'll bump the reqs once more before merging, treating also some open PRs.